### PR TITLE
fixed notifications to chat, sending with postJsonAsync

### PR DESCRIPTION
**Pull request by fchyla**
fix for issue #8  
tested on RocketChat 0.62.0

### CODE REVIEW

File: Notification/RocketChat.php [hunk 1]
```diff
@@ -89,8 +89,8 @@ public function getMessage(array $project, $eventName, array $eventData)
 
         return array(
             'text' => $message,
-            'username' => 'Kanboard',
-            'icon_url' => 'https://kanboard.net/assets/img/favicon.png',
+#            'username' => $author,
+#           'icon_url' => 'https://kanboard.net/assets/img/favicon.png',
```
Comment (fguillot) [inline Notification/RocketChat.php]: Why do you commit commented code?

**Review by fguillot — CHANGES_REQUESTED** [review]

**Comment by fchyla** [comment]
@fguillot commented code removed

File: Notification/RocketChat.php [hunk 2]
```diff
@@ -107,11 +105,12 @@ public function getMessage(array $project, $eventName, array $eventData)
     private function sendMessage($webhook, $channel, array $project, $eventName, array $eventData)
     {
         $payload = $this->getMessage($project, $eventName, $eventData);
-
+
```
Comment (fguillot) [inline Notification/RocketChat.php]: you don't need to commit whitespaces :)

**Review by fguillot — CHANGES_REQUESTED** [review]
The unit tests need to be updated as well. The build is broken.

**Comment by fguillot** [comment]
Are you going to fix the unit tests as well? Let me know if you don't know how to do it.

**Comment by fchyla** [comment]
Apologies for the late reply.
I do not know how to do that. If you could help that would be great.

If I understand correctly Test/RocketChatTest.php has to be rewritten to work with the new JSON message format. I do not have enough knowledge to do it help will be appreciated.

**Comment by fchyla** [comment]
If I understand correctly Test/RocketChatTest.php has to be rewritten to work with the new JSON message format. I do not have enough knowledge to do it help will be appreciated.

**Comment by fguillot** [comment]
PR merged manually.

**Pull request closed without merging**